### DISCUSSION TITLE
Presentation to resonance-mao.be in Brussels Friday 12th of January

### DISCUSSION THREAD

**Discussion by Utopiah** [cardonabits/haxo-hw#67] [Ideas]
Hi there, assuming https://github.com/cardonabits/haxo-hw/issues/61 somehow get sorted I'd like to share my experience.

**Comment by jcard0na** [comment]
Will it be recorded or streamed?

**Comment by Utopiah** [comment]
No it's informal. In fact I just learned there is a https://vcvrack.com workshop this Friday so I'll probably postpone, getting more time to have the RPi Zero working so that I can use MIDI for LMMS and WebMIDI for https://github.com/cardonabits/haxo-hw/discussions/68
